go.mod

```diff
@@ -2,6 +2,8 @@ module kubevirt.io/ssp-operator
 
 go 1.23.6
 
+toolchain go1.23.8
```

Comment:
ranodmly stumbled across this;
are you sure this is the way to go? folks have landed on vague x.y.0 in kubevirt/cdi/k8s following heavy considerations
https://github.com/kubevirt/containerized-data-importer/pull/3496
this allows the host toolchain to take over, and makes the project more importable
/cc @dominikholler

Reply:
@akalenyu thanks for the valuable hint, I agree!